src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -699,8 +704,8 @@ public void start()
 
     private void startClientTransports()
     {
-        String nativeFlag = System.getProperty("cassandra.start_native_transport");
-        if ((nativeFlag != null && Boolean.parseBoolean(nativeFlag)) || (nativeFlag == null && DatabaseDescriptor.startNativeTransport()))
+        String nativeFlag = START_NATIVE_TRANSPORT.getString();
+        if ((Boolean.parseBoolean(nativeFlag)) || (nativeFlag == null && DatabaseDescriptor.startNativeTransport()))
```

Comment:
can't we use `if (START_NATIVE_TRANSPORT.getBoolean() || (nativeFlag == null &&  DatabaseDescriptor.startNativeTransport()))` ?

Also, the parenthesis around `Boolean.parseBoolean(nativeFlag)` are redundant, no?

Reply:
You're right.
Fixed.